pkg/sideinputs/manager/manager_test.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2022 The Numaproj Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manager
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+
+	dfv1 "github.com/numaproj/numaflow/pkg/apis/numaflow/v1alpha1"
+)
+
+func Test_resolveCron(t *testing.T) {
+	f := func() { fmt.Println("hello") }
+
+	t.Run("valid cron schedule", func(t *testing.T) {
+		trigger := &dfv1.SideInputTrigger{
+			Schedule: pointer.String("*/5 * * * *"),
```

Comment:
https://pkg.go.dev/github.com/robfig/cron#hdr-CRON_Expression_Format
`using 6 space-separated fields` I think the package needs [6 fields](https://github.com/robfig/cron/blob/v1.2.0/parser_test.go#L126) instead of [5 fields](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-syntax)

I see tests in https://github.com/numaproj/numaflow/pull/866 also uses 5 fields..

Reply:
we can merge this in and I will update the tests in the next PR

Reply:
perhaps we can merge to a different branch and fix tests in that branch and then merge to main?

Reply:
ignore above :-) I was looking at v1.2.0 while we use v3.